pallets/liquid-staking/src/lib.rs

```diff
@@ -1813,6 +1852,63 @@ pub mod pallet {
             Ok(inflate_liquid_amount)
         }
 
+        #[require_transactional]
+        fn do_fast_match_unstake(unstaker: &T::AccountId) -> DispatchResult {
+            FastUnstakeRequests::<T>::try_mutate_exists(unstaker, |b| -> DispatchResult {
+                if b.is_none() {
+                    return Ok(());
+                }
+                let request_liquid_amount =
+                    b.take()
+                        .expect("Could not be none, qed;")
+                        .min(T::Assets::reducible_balance(
+                            Self::liquid_currency()?,
+                            unstaker,
+                            false,
+                        ));
+
+                let available_liquid_amount =
+                    Self::staking_to_liquid(Self::matching_pool().total_stake_amount.free()?)
```

Comment:
we probably dont need this conversion

Reply:
we need to convert liquid_amount to staking_amount or convert vice versa, so they can compare with each other.